### DISCUSSION TITLE
Keycloak cookies change SameSite attribute to Lax or Strict

### DISCUSSION THREAD

**Discussion by IvanTopic** [keycloak/keycloak#17117] [Q&A]
Hi,

is it possible to change SameSite attribute to something other than "none"?

I've tried using various [Quarkus configuration](https://quarkus.io/guides/all-config) options in the quarkus.properties file that set this attribute with no luck.

Regards.

**Comment by jonkoops** [comment]
Related discussion https://github.com/keycloak/keycloak/pull/16747
